packages/react/src/components/Tile/Tile-story.js

```diff
@@ -165,7 +166,9 @@ export const Expandable = () => (
     <TileBelowTheFoldContent>
       <div style={{ height: '400px' }}>
         Below the fold content here
-        <TextInput id="test2" invalidText="A valid value is required" />
+        <Layer>
+          <TextInput id="test2" invalidText="A valid value is required" />
+        </Layer>
```

Comment:
We'll need to add this to the `next` story as well, not currently showing correctly in the preview

Reply:
Thanks for the catch! Updated